.github/scripts/check_advanced_requirement.sh

```
@@ -0,0 +1,48 @@
+  MSG="Hi @$ASSIGNEE, I cannot assign you to this issue yet.
+
+**Why?**
+Advanced issues involve high-risk changes to the core codebase. They require significant testing and can impact automation and CI behavior. Because of this, we require developers to demonstrate familiarity with the repo first.
+
+**Requirement:**
+- Complete at least **1** 'good first issue' (You have: **$GFI_COUNT**)
+- Complete at least **1** 'intermediate' issue (You have: **$INT_COUNT**)
+
+Please feel free to pick up an [intermediate issue](https://github.com/$REPO/labels/intermediate) to get started! We look forward to your contributions."
+
+  gh issue comment $ISSUE_NUMBER --repo $REPO --body "$(printf "$MSG")"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)